test/boolean_test.cpp

```diff
@@ -898,6 +898,7 @@ TEST(Boolean, Sweep) {
   }
 
   Manifold shape = Manifold::BatchBoolean(result, OpType::Add);
+  EXPECT_EQ(shape.NumTri(), 4400);
```

Comment:
This is checking 3, rather than 2. I'm willing to try it, but I'm guessing this will need updates with future changes. If I'm wrong, so much the better!

Reply:
Indeed, this is quite sensitive to the behavior of the decimator and many other stuff.
I guess I should probably put the deterministic tests elsewhere, and just run the samples 10 times or something to test if they have different results